core/src/draw_target/mod.rs

```diff
@@ -423,3 +423,34 @@ pub trait DrawTarget: Dimensions {
         self.fill_solid(&self.bounding_box(), color)
     }
 }
+
+impl<T> DrawTarget for &mut T
+where
+    T: DrawTarget,
+{
+    type Color = T::Color;
+
+    type Error = T::Error;
+
+    fn draw_iter<I>(&mut self, pixels: I) -> Result<(), Self::Error>
+    where
+        I: IntoIterator<Item = Pixel<Self::Color>>,
+    {
+        (**self).draw_iter(pixels)
```

Comment:
Just a stylistic choice, but IMO using the `T` type explicitly makes these statements easier to understand:
```suggestion
        T::draw_iter(self, pixels)
```

Reply:
I don't really have a strong preference here, so I'm fine with that.